src/main.ts

```diff
@@ -35,11 +35,13 @@ async function run(): Promise<void> {
     const baseDir = path.join(process.cwd(), core.getInput('cwd') || '')
     const readmePath = path.join(
       baseDir,
-      core.getInput('readmePath') || 'README.md'
+      core.getInput('readme_path') || 'README.md'
     )
     const readmeContent = await fs.readFile(readmePath, {
       encoding: 'utf-8'
     })
+    const headingLevel: number = +core.getInput('heading_level') || 1
```

Comment:
[comment] Rather than using JavaScript's coercion, I prefer to be explicit, so I'm going to swap this out for `parseInt(core.getInput('heading_level'), 10) || 1` which is going to do the same thing.